Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -85,8 +85,7 @@ open class ChatChannelListVC: _ViewController,
         
         // Set the Controller on the ViewController
         chatChannelListVC.controller = controller
-        chatChannelListVC.setUp()
```

Comment:
This seems legit, the `setUp` get invoked automatically as part of the `ChatChannelListVC's` view lifecycle 🤔

@adamrushy can we make this change or there's a reason to have it?

Reply:
Yeah, I need to check this... I specifically added because there was a bug with delegates not being assigned if I remember rightly 🤔

Reply:
Okay, I tested this and is all working as expected so we can proceed. I also checked by using the Storyboard implementation and it's all working as expected

`let chatList = DemoChannelListVC.make(with: controller, storyboard: UIStoryboard(name: "Main", bundle: .main), storyboardId: "ChannelList")`